### PR TITLE
Added recipe for biorbd

### PR DESCRIPTION
**Pull request by pariterre**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pariterre** [comment]
@conda-forge/core
Hello there, I don't understand about this extra lines issue...

File: recipes/biorbd/bld.bat [hunk 1]
```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+cmake ../^
+    -G"Visual Studio 15 2017 Win64"^
```
Comment (isuruf) [inline recipes/biorbd/bld.bat]: Use `-G"%CMAKE_GENERATOR%"`

File: recipes/biorbd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  description: |
+    Biomechanical add-ons to the RigidBody Dynamics Library .
+  doc_url: https://github.com/pyomeca/biorbd/
+  dev_url: https://github.com/pyomeca/biorbd/
+
+extra:
+  recipe-maintainers:
+    - pariterre
+
```
Comment (isuruf) [inline recipes/biorbd/meta.yaml]: Remove this line. That's what the linter is complaining about

**Comment by pariterre** [comment]
@conda-forge/staged-recipes 
Hello! I am trying to integrate the tests. But it doesn't work. For instance, if I skip osx test but run it on linux, it works on osx, but fails on linux… same thing the other way around.

The error seems to be :  Unsatisfiable dependencies for platform linux-64: {"libboost[version='<0']"} while testing. Do you know where it is from?

thx :)

**Comment by xhochy** [comment]
@pariterre `libboost` is the boost package from `anaconda/defaults` which conflicts with `boost-cpp` on `conda-forge`. Only one of both can be installed and for `conda-forge` recipes `boost-cpp` should always be used. I would guess that one of your `run` dependencies is being installed from `anaconda/defaults` and this is in conflict with a `conda-forge` package.

**Comment by pariterre** [comment]
@xhochy 
I feel that it should be okay.. here are the dependencies as written in the meta file : 
 ```
requirements:
  build:
    - {{ compiler('cxx') }}
    - cmake
    - make  # [unix]
  host:
    - boost-cpp
    - eigen
    - lapack
    - rbdl
    - dlib-cpp
    - python
    - numpy
    - swig
  run:
    - boost-cpp
    - eigen
    - rbdl
    - dlib
    - python
    - {{ pin_compatible('numpy') }}
```

Do you see anything suspicious?

**Comment by pariterre** [comment]
@xhochy 
dlib-cpp... 
I ping you if this doesn't work

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
Ready to review!

**Comment by scopatz** [comment]
Thanks @pariterre!

**Pull request merged**